### PR TITLE
add api ui to compose

### PR DESCRIPTION
**Pull request by kade-ddnkv**

### CODE REVIEW

File: docker-compose.override.yml [hunk 1]
```diff
@@ -92,6 +92,19 @@ services:
       - ./grafana/dashboards:/etc/grafana/dashboards
     networks:
       privnet:
+  
+  ui:
```
Comment (pirog-spb) [inline docker-compose.override.yml]: `ui` -> `edgecom-upf-ui`

File: docker-compose.override.yml [hunk 2]
```diff
@@ -92,6 +92,19 @@ services:
...
+    container_name: ui
+    image: ghcr.io/edgecomllc/eupf-api-ui:74adead324e00eb83f97adfec40c7188c5f57a4a
```
Comment (pirog-spb) [inline docker-compose.override.yml]: BTW `eupf-ui` would be much better name

File: docker-compose.override.yml (hunk 1 above)
Comment (kade-ddnkv) [reply docker-compose.override.yml]: Done

File: docker-compose.override.yml (hunk 2 above)
Comment (kade-ddnkv) [reply docker-compose.override.yml]: Done

**Review by pirog-spb — APPROVED** [review]

**Pull request merged**